arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -91,6 +91,17 @@ def start_pcp(
             if "error" in pmlogconf_return[0]:
                 return pmlogconf_return
 
+        # Import the pmrep configuration file, if provided
+        if params.pmrep_conf:
+            pmrep_conf_path = "pmrep.conf"
+            print("Using provided pmrep configuration file")
+            f = open(pmrep_conf_path, "w")
+            f.write(params.pmrep_conf)
+            f.close
+        else:
+            print("Using default pmrep configuration directory")
+            pmrep_conf_path = "/etc/pcp/pmrep"
```

Comment:
Reporting the fact that you're using the "default" file is good, but it would be better if you also reported what that file was....